### ISSUE TITLE
hlsl.additionalIncludeDirectories "." does not work

### ISSUE THREAD

**Issue by OndrejPetrzilka** [tgjones/HlslTools#91]
Base.h
```
// Content does not matter
```
shadertoolsconfig.json
```
{
    "root": true,
    "hlsl.preprocessorDefinitions": { },
    "hlsl.additionalIncludeDirectories": [ "." ]
}
```
Sprite/Sprite.h
```
#include "Base.h" // This cannot be found
```

Doc says "." is directory which contains "shadertoolsconfig.json" file...so everything should work
It used to work with previous version (168 or so)
When I change it to `#include "../Base.h"` it works, but that's not what I want, I like to reference files relative to root

Version: 1.1.228, VS2015 Community

**Comment by OndrejPetrzilka** [comment]
This seems to work in 1.1.241, I'll check more cases and close the issue if it's no longer a problem.

**Comment by OndrejPetrzilka** [comment]
Works very well, closing, probably fixed somewhere between 1.1.228 and 1.1.241
